web/viewer.js

```diff
@@ -1498,10 +1492,26 @@ function webViewerInitialized() {
     PDFViewerApplication.open(file, 0);
   }
 //#endif
+//#if !PRODUCTION
+  if (true) {
+    return;
+  }
+//#endif
+//#if (FIREFOX || MOZCENTRAL)
+  PDFViewerApplication.setTitleUsingUrl(file);
+  PDFViewerApplication.initPassiveLoading();
+
+  // Clicking on either of the viewBookmark buttons throws a SecurityError
+  // for file:// URLs, hence we hide the buttons since they won't work.
+  if (file && file.lastIndexOf('file:', 0) === 0) {
+    document.getElementById('viewBookmark').classList.add('hidden');
+    document.getElementById('secondaryViewBookmark').classList.add('hidden');
+  }
+//#endif
 //#if CHROME
-//if (file) {
-//  ChromeCom.openPDFFile(file);
-//}
```

Comment:
Why did you change this? `ChromeCom` is only for the Chrome extension, so this should only be enabled when the Chrome extension is build, and not when the PDF viewer is loaded without building.